roles/manage_cnp/defaults/main.yml

```diff
@@ -0,0 +1,134 @@
+---
+# defaults file for manage_cnp
+# configuration variable for which
+# task for execute within role
+#cnp_task: "drop-db"
+# Variables
+host: localhost
+# CloudNativePG Namespace
+cnpg_namespace: default
+# CloudNativePG Pod Name
+pod_name: cnp-sandbox-1
+# Roles
+db_role: edb_test_user
+db_role_password: admin
+# Databases
+db_name: edb
+# Tables
+tbl_name: films
+# Schemas
+schema_name: edb_schema
+# Views
+view_name: edb_view_notnullfilms
+# Indexes
+index_name: edb_index_film_name
+# Sequences
+sequence_name: edb_seq_name
+# Procedures
+procedure_name: edb_procedure_insert_film
+# Functions
+function_name: edb_get_films_count
+# Triggers
+trigger_function_name: log_check_filmname_changes
+trigger_name: edb_films_trigger
+# Types
+type_tbl_name: courses
+type_name: edb_type_enum
+
+# SQL Statements
+# Roles
+add_role_sql: |
+     CREATE ROLE {{ db_role }} LOGIN PASSWORD '{{ db_role_password  }}';
+drop_role_sql:
+  DROP ROLE {{ db_role }};
```

Comment:
This could be all on one line, with quotes around the command `drop_role_sql: "DROP ROLE {{ db_role }};"`

Reply:
updated